lib/server.js

```diff
@@ -346,15 +347,15 @@ Server.prototype._respond = function(txn, res, cb) {
 Server.prototype._exchange = function(type, req, res, cb) {
   var stack = this._exchanges
     , idx = 0;
-  
+
   function next(err) {
     if (err) { return cb(err); }
-    
+
     var layer = stack[idx++];
     if (!layer) { return cb(); }
-    
+
```

Comment:
@loay I see, probably we should have added `eslint` which could take care of all of these things before...